### PR TITLE
Tell users to enable reaction updates

### LINKED ISSUE #990: Reactions documentation: Please add information about enabling them in "allowed_updates" field.
[https://grammy.dev/guide/reactions](https://grammy.dev/guide/reactions)

It would be helpful to mention that you need to enable these updates in the `bot.start` function, like this:
```
bot.start({
  allowed_updates: ["message", "message_reaction", "message_reaction_count"]
});
```
Otherwise, they wouldn't work.

Thank you anyway for the awesome library and your website!

### PR DESCRIPTION
**Pull request by KnorpelSenf**
Closes #990.

### CODE REVIEW

**Comment by KnorpelSenf** [comment]
Approved in https://github.com/grammyjs/website/issues/990#issuecomment-1938738998, @roj1512 please nitpick

**Review by roj1512 — APPROVED** [review]

File: site/docs/id/guide/reactions.md [hunk 1]
```diff
@@ -46,6 +46,30 @@ Terdapat beberapa cara untuk menangani update berupa reaksi.
...
+```
+
+::: tip Mengaktifkan Semua Jenis Update
+Untuk menerima semua jenis update, import `API_CONSTANTS` dari grammY lalu cantumkan di bagian `allowed_updates`.
+
+```ts
+allowed_updates: API_CONSTANTS.ALL_UPDATE_TYPES;
+```
+
+Jangan lupa untuk membaca [Referensi API](https://deno.land/x/grammy@v1.20.4/mod.ts?s=ApiConstants#prop_ALL_UPDATE_TYPES)-nya.
```
Comment (quadratz) [inline site/docs/id/guide/reactions.md:65]:
```suggestion
Jangan lupa untuk membaca [Referensi API](https://deno.land/x/grammy/mod.ts?s=ApiConstants#prop_ALL_UPDATE_TYPES)-nya.
```

File: site/docs/uk/guide/reactions.md [hunk 2]
```diff
@@ -46,6 +46,30 @@ await bot.api.setMessageReaction(chat_id, message_id, "💯");
...
+
+::: tip Увімкнення всіх оновлень
+Ви також можете імпортувати `API_CONSTANTS` з grammY, в подальшому використавши змінну як
+
+```ts
+allowed_updates: API_CONSTANTS.ALL_UPDATE_TYPES;
+```
+
+для того, щоб отримувати всі оновлення.
+Обов'язково перегляньте [довідку API](https://deno.land/x/grammy@v1.20.4/mod.ts?s=ApiConstants#prop_ALL_UPDATE_TYPES).
```
Comment (quadratz) [inline site/docs/uk/guide/reactions.md:66]:
```suggestion
Обов'язково перегляньте [довідку API](https://deno.land/x/grammy/mod.ts?s=ApiConstants#prop_ALL_UPDATE_TYPES).
```

File: site/docs/uk/guide/reactions.md [hunk 3]
```diff
@@ -46,6 +46,30 @@ await bot.api.setMessageReaction(chat_id, message_id, "💯");
 У приватних і групових чатах ваш бот отримає оновлення `message_reaction`, якщо користувач змінить свою реакцію на повідомлення.
 У каналах (або автоматично перенаправлених дописах каналів у групах) ваш бот отримає оновлення `message_reaction_count`, яке показує лише загальну кількість реакцій, але не розкриває, хто саме відреагував.
 
+Обидва типи реакцій потрібно увімкнути, перш ніж ви зможете їх отримувати.
+Наприклад, за допомогою вбудованого тривалого опитування ви можете увімкнути їх таким чином:
```
Comment (deptyped) [inline site/docs/uk/guide/reactions.md:50]:
```suggestion
Наприклад, для вбудованого тривалого опитування ви можете увімкнути їх таким чином:
```

**Review by quadratz — CHANGES_REQUESTED** [review]

File: site/docs/uk/guide/reactions.md [hunk 4]
```diff
@@ -92,9 +116,13 @@ grammY надає вам можливість ще більше відфільт
...
 // Оновлення, в яких поточна реакція містить принаймні один емодзі.
-bot.on("message_reaction:new_reaction:emoji", (ctx) => {/* ... */});
+bot.on("message_reaction:new_reaction:emoji", (ctx) => {
+  /* ... */
+});
 // Оновлення, в яких попередня реакція містила принаймні один користувацький емодзі.
-bot.on("message_reaction:old_reaction:custom_emoji", (ctx) => {/* ... */});
+bot.on("message_reaction:old_reaction:custom_emoji", (ctx) => {
+  /* ... */
+});
```
Comment (deptyped) [inline site/docs/uk/guide/reactions.md:125]: Please revert these changes

File: site/docs/uk/guide/reactions.md [hunk 5]
```diff
@@ -46,6 +46,30 @@ await bot.api.setMessageReaction(chat_id, message_id, "💯");
...
+allowed_updates: API_CONSTANTS.ALL_UPDATE_TYPES;
+```
+
+для того, щоб отримувати всі оновлення.
+Обов'язково перегляньте [довідку API](https://deno.land/x/grammy@v1.20.4/mod.ts?s=ApiConstants#prop_ALL_UPDATE_TYPES).
+:::
+
+[Плагін для конкурентності (runner)](../plugins/runner.md#просунуті-параметри) і `setWebhhok` мають схожі способи вказати `allowed_udpates`.
+
+Тепер, коли ваш бот може отримувати оновлення реакцій, давайте подивимося, як він з ними впорається!
```
Comment (deptyped) [inline site/docs/uk/guide/reactions.md:71]:
```suggestion
Тепер, коли ваш бот може отримувати оновлення реакцій, давайте подивимося, як він може їх обробляти!
```

**Review by deptyped — CHANGES_REQUESTED** [review]

File: site/docs/guide/reactions.md [hunk 6]
```diff
@@ -46,6 +46,30 @@ There are a few different ways to handle updates about reactions.
...
+
+```ts
+allowed_updates: API_CONSTANTS.ALL_UPDATE_TYPES;
+```
+
+in order to receive all updates.
+Be sure to check out the [API reference](https://deno.land/x/grammy@v1.20.4/mod.ts?s=ApiConstants#prop_ALL_UPDATE_TYPES).
+:::
+
+[grammY runner](../plugins/runner.md#advanced-options) and `setWebhhok` have similar ways to specify `allowed_udpates`.
```
Comment (deptyped) [inline site/docs/guide/reactions.md:69]:
```suggestion
[grammY runner](../plugins/runner.md#advanced-options) and `setWebhook` have similar ways to specify `allowed_updates`.
```

File: site/docs/id/guide/reactions.md [hunk 7]
```diff
@@ -46,6 +46,30 @@ Terdapat beberapa cara untuk menangani update berupa reaksi.
...
+:::
+
+Di [grammY runner](../plugins/runner.md#opsi-tingkat-lanjut) dan `setWebhhok`, `allowed_udpates` juga bisa diatur dengan cara yang sama.
```
Comment (deptyped) [inline site/docs/id/guide/reactions.md:68]:
```suggestion
Di [grammY runner](../plugins/runner.md#opsi-tingkat-lanjut) dan `setWebhook`, `allowed_updates` juga bisa diatur dengan cara yang sama.
```

File: site/docs/uk/guide/reactions.md [hunk 8]
```diff
@@ -46,6 +46,30 @@ await bot.api.setMessageReaction(chat_id, message_id, "💯");
...
+
+```ts
+allowed_updates: API_CONSTANTS.ALL_UPDATE_TYPES;
+```
+
+для того, щоб отримувати всі оновлення.
+Обов'язково перегляньте [довідку API](https://deno.land/x/grammy@v1.20.4/mod.ts?s=ApiConstants#prop_ALL_UPDATE_TYPES).
+:::
+
+[Плагін для конкурентності (runner)](../plugins/runner.md#просунуті-параметри) і `setWebhhok` мають схожі способи вказати `allowed_udpates`.
```
Comment (deptyped) [inline site/docs/uk/guide/reactions.md:69]:
```suggestion
[Плагін для конкурентності (runner)](../plugins/runner.md#просунуті-параметри) і `setWebhook` мають схожі способи вказати `allowed_updates`.
```

**Review by deptyped — CHANGES_REQUESTED** [review]

File: site/docs/guide/reactions.md (hunk 6 above)
Comment (quadratz) [reply site/docs/guide/reactions.md:69]: good eye

File: site/docs/uk/guide/reactions.md (hunk 3 above)
Comment (niusia-ua) [reply site/docs/uk/guide/reactions.md:50]:
```suggestion
Наприклад, для вбудованого тривалого опитування ви можете увімкнути їх так:
```

File: site/docs/uk/guide/reactions.md [hunk 9]
```diff
@@ -46,6 +46,30 @@ await bot.api.setMessageReaction(chat_id, message_id, "💯");
...
+Наприклад, за допомогою вбудованого тривалого опитування ви можете увімкнути їх таким чином:
+
+```ts
+bot.start({
+  allowed_updates: ["message", "message_reaction", "message_reaction_count"],
+});
+```
+
+::: tip Увімкнення всіх оновлень
+Ви також можете імпортувати `API_CONSTANTS` з grammY, в подальшому використавши змінну як
```
Comment (niusia-ua) [inline site/docs/uk/guide/reactions.md:59]:
```suggestion
Ви також можете імпортувати `API_CONSTANTS` з grammY і вказати
```

File: site/docs/uk/guide/reactions.md (hunk 4 above)
Comment (LWJerri) [reply site/docs/uk/guide/reactions.md:125]: I can't, because it's result of `npm run docs:fmt`.

File: site/docs/guide/reactions.md (hunk 6 above)
Comment (KnorpelSenf) [reply site/docs/guide/reactions.md:69]: Indeed, not even Roj found this

File: site/docs/uk/guide/reactions.md [hunk 10]
```diff
@@ -92,9 +116,13 @@ grammY надає вам можливість ще більше відфільт
 
 ```ts
 // Оновлення, в яких поточна реакція містить принаймні один емодзі.
-bot.on("message_reaction:new_reaction:emoji", (ctx) => {/* ... */});
+bot.on("message_reaction:new_reaction:emoji", (ctx) => {
+  /* ... */
+});
```
Comment (LWJerri) [inline site/docs/uk/guide/reactions.md:121]:
```suggestion
bot.on("message_reaction:new_reaction:emoji", (ctx) => {/* ... */});
```

File: site/docs/uk/guide/reactions.md (hunk 4 above)
Comment (LWJerri) [inline site/docs/uk/guide/reactions.md:125]:
```suggestion
bot.on("message_reaction:old_reaction:custom_emoji", (ctx) => {/* ... */});
```

**Review by niusia-ua — APPROVED** [review]

**Review by LWJerri — APPROVED** [review]

**Review by deptyped — APPROVED** [review]

File: site/docs/guide/reactions.md [hunk 11]
```diff
@@ -46,6 +46,30 @@ There are a few different ways to handle updates about reactions.
...
+
+```ts
+allowed_updates: API_CONSTANTS.ALL_UPDATE_TYPES;
+```
+
+in order to receive all updates.
+Be sure to check out the [API reference](https://deno.land/x/grammy/mod.ts?s=ApiConstants#prop_ALL_UPDATE_TYPES).
+:::
+
+[grammY runner](../plugins/runner.md#advanced-options) and `setWebhook` have similar ways to specify `allowed_updates`.
```
Comment (quadratz) [inline site/docs/guide/reactions.md:69]:
```suggestion
[grammY runner](../plugins/runner#advanced-options) and `setWebhook` have similar ways to specify `allowed_updates`.
```

File: site/docs/id/guide/reactions.md [hunk 12]
```diff
@@ -46,6 +46,30 @@ Terdapat beberapa cara untuk menangani update berupa reaksi.
...
+Untuk menerima semua jenis update, import `API_CONSTANTS` dari grammY lalu cantumkan di bagian `allowed_updates`.
+
+```ts
+allowed_updates: API_CONSTANTS.ALL_UPDATE_TYPES;
+```
+
+Jangan lupa untuk membaca [Referensi API](https://deno.land/x/grammy/mod.ts?s=ApiConstants#prop_ALL_UPDATE_TYPES)-nya.
+:::
+
+Di [grammY runner](../plugins/runner.md#opsi-tingkat-lanjut) dan `setWebhook`, `allowed_updates` juga bisa diatur dengan cara yang sama.
```
Comment (quadratz) [inline site/docs/id/guide/reactions.md:68]:
```suggestion
Di [grammY runner](../plugins/runner#opsi-tingkat-lanjut) dan `setWebhook`, `allowed_updates` juga bisa diatur dengan cara yang sama.
```

File: site/docs/uk/guide/reactions.md [hunk 13]
```diff
@@ -46,6 +46,30 @@ await bot.api.setMessageReaction(chat_id, message_id, "💯");
...
+
+```ts
+allowed_updates: API_CONSTANTS.ALL_UPDATE_TYPES;
+```
+
+для того, щоб отримувати всі оновлення.
+Обов'язково перегляньте [довідку API](https://deno.land/x/grammy/mod.ts?s=ApiConstants#prop_ALL_UPDATE_TYPES).
+:::
+
+[Плагін для конкурентності (runner)](../plugins/runner.md#просунуті-параметри) і `setWebhook` мають схожі способи вказати `allowed_updates`.
```
Comment (quadratz) [inline site/docs/uk/guide/reactions.md:69]:
```suggestion
[Плагін для конкурентності (runner)](../plugins/runner#просунуті-параметри) і `setWebhook` мають схожі способи вказати `allowed_updates`.
```

**Review by quadratz — CHANGES_REQUESTED** [review]
Remove file extension from local path

**Review by quadratz — APPROVED** [review]

**Comment by KnorpelSenf** [comment]
whoop whoop
